Tests/Test_BSP_Timer.c

```diff
@@ -0,0 +1,38 @@
+#include "BSP_Timer.h"
+
+/** 
+ * @brief   Test function to see if the first Timer works
+ * @param   none
+ * @return  none (will print a string on console)
+ */  
+void functionOne()
+{
+  printf("First Function worked");
+
+}
+
+/** 
+ * @brief   Test function to see of the second Timer works 
+ * @param   none
+ * @return  none (will print a string on console)
+ */  
+void functionTwo()
+{
+  printf("Second Function worked");
+}
+
+
+int main()
+{
+    void (*functOnepointer)= &functionOne;
+    void (*functTwopointer)= &functionTwo;
```

Comment:
You don't have to make these variables. You can just pass `functionOne` and `functionTwo` to the Init function